src/eddymotion/estimator.py

```diff
@@ -105,45 +104,32 @@ def fit(
                         # generate a synthetic dw volume for the test gradient
                         predicted = dwmodel.predict(data_test[1])
 
-                        tmpdir = Path(tmpdir)
-                        moving = tmpdir / "moving.nii.gz"
-                        fixed = tmpdir / "fixed.nii.gz"
-                        _to_nifti(data_test[0], dwdata.affine, moving)
-                        _to_nifti(
-                            predicted,
-                            dwdata.affine,
-                            fixed,
-                            clip=reg_target_type == "dwi",
```

Comment:
Looks like the clipping depends on the type of the fixed image (whether it's a b=0 or a diffusion-weighted volume). I guess we need to reincorporate clipping somehow.